src/ask-password/ask-password.c

```diff
@@ -155,6 +165,18 @@ static int parse_argv(int argc, char *argv[]) {
                         assert_not_reached("Unhandled option");
                 }
 
+        if (!emoji || streq(emoji, "auto"))
```

Comment:
i'd really accept the empty string here too, as equivalent to "auto". i.e.:

```c
if (isempty(emoji) || streq(emoji, "auto")) …
```

we usually accept it like that.